### ISSUE TITLE
Feature request: track/show more quality metrics

### ISSUE THREAD

**Issue by nathan-at-least** [rust-lang/docs.rs#2153]
When I'm searching for crates, I poke around trying to get a quick impression of "quality" or more precisely "maintenance reliability". What I really want to know are answers to questions like:

- how likely is it this crate has addressed low-hanging fruit bugs for its use case?
- if there's a bug, flaw, or I request a new feature, how likely is it devs are responsive and will make a new release addressing the issue?
- how many other crates or people rely on this crate?
- do the devs maintain other crates with high or low quality metrics?

Here are some brainstorms of various quality metrics that may be plausible to track/display:

- how many other crates depend on the search result crate?
- how many outstanding `cargo audit` failures does the crate have?
- what's the ratio of opened vs closed issues on Github (or other issue trackers)? (Could be messy to support 3rd party issue trackers.)
- something about release cadence, like "this crate makes regular releases every N months"…
- (from #2151) this crate has `#![deny(missing_docs)]` guaranteeing all pub items at least have documentation.

BTW- I do _not_ recommend aggregating these into some top-level "quality" metric, but instead indicating them in search results independently, since different users may value them differently (and they are all imperfect approximations of the ideal info). If this is too much clutter, there could be a widget to expand these as columns that are hidden by default.

**Comment by nathan-at-least** [comment]
Note: as with all metrics these could be maliciously gamed (e.g. making a bunch of dummy crates that depend on a crate to escalate that number, etc…), so metrics are never a guarantee of quality.

**Comment by syphar** [comment]
Hi, 
thank you for the idea! 

The list of requirements sound an aweful lot like you're trying to discover new crates using the docs.rs search, am I getting this right? 

Some of the information is already available and visible directly on crates.io, or libs.rs. 
And the use-case of "discover new crates" should IMO be either solved by crates.io, or sites like libs.rs ([which also tries to add ranking](https://lib.rs/about)). 

The rustdoc-status or more information could be fetched by the other platforms (similar to shields.io and crates.io, see #2144). 

There is also an open discussion to show open advisories on crates.io ( https://github.com/rust-lang/crates.io/issues/6397 ). 

So to sum it up: 
For now I believe "discover new crates" isn't something that docs.rs should handle. 

Could you elaborate where do the other platforms do not handle your use-case?

**Comment by syphar** [comment]
I see libs.rs already has integrated rustdoc advisories: 
- https://lib.rs/crates/openssl-src/versions
- https://gitlab.com/crates.rs/crates.rs/-/issues/6

**Comment by syphar** [comment]
I'm closing this as "wontfix": 

crate discovery is not a use-case for the docs.rs search, crates.io is already slowly adding functionality for this.
